spiffworkflow-backend/src/spiffworkflow_backend/services/upsearch_service.py

```diff
@@ -0,0 +1,13 @@
+import os
+
+
+class UpsearchService:
+    @classmethod
+    def upsearch_locations(cls, location: str) -> list[str]:
```

Comment:
Can this be `process_model_identifier` or `process_group_identifier` instead of location? So it's easier to understand what needs to be passed in.

Reply:
I was thinking of it more abstractly, like give any location and get the locations to upsearch for it, but in practice so far we are only passing in process_model_identifiers so will rename it to that.